tests/agents_map.py

```diff
@@ -0,0 +1,60 @@
+"""
+Generate agent parameterization for pytest.
+"""
+import os
+import sys
+from loguru import logger
+from munch import Munch
+from configuration import elasticsearch
```

Comment:
I'm not sure the location of the` tests/agents_map.py` file is ideal. The tests folder serves as the root for the tests project and should contain all relevant project files and configurations. A better approach would be to store this file under the `tests/commonlib` path.